langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -333,6 +473,13 @@ public isolated function elementChildren(xml x, string? nm = ()) returns xml<Ele
 #
 # Each item is represented as a singleton value.
 #
+# ```ballerina
+# xml books = xml `<book>Java</book><book>Ballerina</book>`;
+# books.'map(function (xml xmlContent) returns xml {
```

Comment:
```suggestion
# books.map(function (xml xmlContent) returns xml {
```

Reply:
@sanjana sync to the PR sent to 3.x

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/17b6885a7702ae6cf2337cce0d1e39daa85cc83d